app/Http/Controllers/Eportalcontroller.php

```diff
@@ -85,33 +98,32 @@ public function store(Request $request)
             $file->save();
             if($request->abc != null)
             {
-                return redirect('/show?abc='.$file->parent_directory)->with("success","Successfully Uploaded");
+                return redirect('/files'.'/'. $file->parent_directory)->with("success","Folder Created");
```

Comment:
With PSR2 guidelines, you need to follow camelCasing
```suggestion
                return redirect('/files'.'/'. $file->parentDirectory)->with("success","Folder Created");
```